1-Vanilla/scripts/todo_list.js

```diff
@@ -0,0 +1,130 @@
+let nextId = 0;
+
+function createButton(id, type, cb) {
+  let button = document.createElement("div");
+  let icon = "&#xe145;";
+
+  switch (type) {
+    case "edit":
+      icon = "&#xe3c9;";
+      break;
+    case "remove":
+      icon = "&#xe14c;";
+      break;
+    case "check":
+      icon = "&#xe876;";
+      break;
+    default:
+      break;
+  }
+
+  button.classList.add("btn");
+  button.classList.add(type + "-btn");
+  button.id = type + "-btn-" + id;
+  button.innerHTML = '<span class="material-icons md-36">' + icon + "</span>";
```

Comment:
prefer the usage of the DOM api to create & append child elements rather than using the innerHTML property
makes the code more readable and maintainable.